soccer/NewRefereeModule.cpp

```diff
@@ -283,6 +283,8 @@ void NewRefereeModule::updateGameState(bool blueTeam) {
             break;
         case Command::FORCE_START:
             _state.gameState.state = GameState::Playing;
+            _state.gameState.ourRestart = false;
+            _state.gameState.restart = GameState::None;
```

Comment:
That should be the only place that needed the restart variable to change, the rest of the time it gets handled outside of the refmodule.